buildpipeline/pipeline.json

```diff
@@ -126,6 +126,7 @@
           "Parameters": {
             "PB_AdditionalBuildArguments": "/p:PortableBuild=true /p:SkipTests=true",
             "PB_TargetArchitecture": "arm",
+            "PB_DistroRid": "win8-arm",
```

Comment:
If Portable build is being triggered, then "win-arm" and "win-arm64". Otherwise, "win8-arm" and "win10-arm64".